src/libraries/System.Diagnostics.EventLog/src/System/Diagnostics/Reader/EventLogException.cs

```diff
@@ -50,7 +50,7 @@ internal static void Throw(int errorCode)
         public EventLogException() { }
         public EventLogException(string message) : base(message) { }
         public EventLogException(string message, Exception innerException) : base(message, innerException) { }
-        protected EventLogException(int errorCode) { _errorCode = errorCode; }
```

Comment:
Can you please fix formatting here -- braces on their own line, etc.

Also, I believe this should be `HResult = Win32Marshal.MakeHRFromErrorCode(errorCode);`